.changeset/silver-jeans-wash.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Added the changeset to let users know that something has changed in the config validation.